commands.go

```diff
@@ -560,18 +572,86 @@ func getRemovableScheduleJobs(c *config.Config, flags commandLineFlags) (schedul
 	for _, command := range profile.SchedulableCommands() {
 		declared := false
 		for _, s := range schedules {
-			if declared = s.SubTitle == command; declared {
+			if declared = s.CommandName == command; declared {
 				break
 			}
 		}
 		if !declared {
-			schedules = append(schedules, config.NewRemoveOnlyConfig(profile.Name, command))
+			schedules = append(schedules, config.NewEmptySchedule(profile.Name, command))
 		}
 	}
 
 	return scheduler, profile, schedules, nil
 }
 
+func preRunSchedule(ctx *Context) error {
+	if len(ctx.request.arguments) < 1 {
+		return errors.New("run-schedule command expects one argument: schedule name")
+	}
+	scheduleName := ctx.request.arguments[0]
+	if ctx.config.GetVersion() < config.Version02 {
```

Comment:
If we limit it to version 1, version 2 files will no longer support scheduling, don't they?

Reply:
hmmm yeah good point:
In case it takes me another year to finish the second part of the scheduling v2, we might still want to do a release in between 😆